health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/InboxFilterWrapper.js

```
@@ -202,24 +202,27 @@ const InboxFilterWrapper = (props) => {
           }));
 
           const isLastElement = index === state.securityQuestions.length - 1;
+          const questionNumber = parseInt(item.id, 10); 
 
           return (
             <LabelFieldPair
               vertical
               style={{ paddingBottom: isLastElement ? "1rem" : "0" }} 
             >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)